tests/end-to-end/api/00-miscellaneous.js

```diff
@@ -98,4 +98,34 @@ describe('miscellaneous', function() {
 		});
 	});
 
+	describe('[/spotlight]', () => {
```

Comment:
Can we add a test that ensures the `#channel` works as expected? Also, actually ensure the result contains member and room objects instead of the below length that checks for `0`?

I know this was just moved to the correct test file, but since we are adding a new feature to the endpoint I think it would be great to test it.